src/main/java/dev/ai4j/openai4j/LogLevelEnum.java

```diff
@@ -0,0 +1,9 @@
+package dev.ai4j.openai4j;
+
+public enum LogLevelEnum {
```

Comment:
Do you really need this `Enum` prefix? I guess the final client code is cleaner without: `.logRequestsLevel(LogLevelEnum.INFO)`

Reply:
You mean rename it to `LogLevel`?

Reply:
exactly